comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -368,6 +381,10 @@ func (ia *inventoryagent) refreshMetadata() {
 	ia.fetchTraceAgentMetadata()
 	// system-probe ecosystem
 	ia.fetchSystemProbeMetadata()
+	// Fleet
+	ia.fetchFleetMetadata()
+	// HA Agent
+	ia.fetchHaAgentMetadata()
```

Comment:
Please update the documentation and payload example in this package README

Reply:
@hush-hush sure, updated here https://github.com/DataDog/datadog-agent/pull/33215/commits/4672faea33ca7f6998899327d0de430d5ee6ee7d